packages/browser-wallet/src/popup/shared/TokenDetails/TokenDetails.tsx

```diff
@@ -86,6 +86,11 @@ function Nft({ token, balance, contractIndex, subIndex }: TokenProps) {
                 {thumbnail && <Img src={thumbnail.url} alt={name} withDefaults />}
                 {name}
             </h3>
+            {display?.url && (
```

Comment:
There is no limit on the height of the image? Isn't that an issue?

Reply:
I don't think so.. What's worse, to scroll a lot to see the full image, or to not be able to see it due to it being slimmed down to nothing due to a height limit? I prefer the scrolling option, I think? :shrug:

Reply:
I was more thinking about the other data. I guess I just disagree with it being on top :thinking: